addons/website_rating/models/mail_message.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class MailMessage(models.Model):
+    _inherit = 'mail.message'
+
+    rating_value = fields.Float("Rating Value", compute='_compute_rating_value', search='_search_rating_value')
+
+    @api.multi
+    def _compute_rating_value(self):
+        ratings = self.env['rating.rating'].search([('message_id', 'in', self.ids), ('consumed', '=', True)], order='create_date DESC')
+        mapping = dict((r.message_id.id, r.rating) for r in ratings)
+        for message in self:
+            message.rating_value = mapping.get(message.id, 0.0)
+
+    def _search_rating_value(self, operator, value):
+        self.env.cr.execute("""
```

Comment:
I fear it will become quickly hard to perform a search. Once you have a lot of ratings, you will have a domain term like [('id', 'in', [A MAGNIFICIENT ARRAY])]. Don't you think we should store the rating value in order to search efficiently on it ?